src/steps/os/archlinux.rs

```diff
@@ -150,6 +150,40 @@ impl Pacman {
     }
 }
 
+pub struct Pikaur {
+    executable: PathBuf
+}
+
+impl Pikaur {
+    fn get() -> Option<Self> {
+        Some(Self {executable: which("pikaur")?})
+    }
+}
+
+impl ArchPackageManager for Pikaur {
+    fn upgrade(&self, ctx: &ExecutionContext) -> Result<()> {
+        let mut command = ctx.run_type().execute(&self.executable);
+
+        command
```

Comment:
Please add support for `--noconfirm` here as well.

Reply:
Fixed